package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "open-msupply",
   "//": "Main version for the app, should be in semantic version format (any release candidate or test build should be separated by '-' i.e. 1.1.1-rc1 or 1.1.1-test",
-  "version": "2.0.02",
+  "version": "2.0.3",
```

Comment:
```suggestion
  "version": "2.0.03",
```

Reply:
Done --but why do we do this? It's not standard semantic versioning 🤷‍♂️
<img width="611" alt="Screenshot 2024-06-05 at 2 30 17 PM" src="https://github.com/msupply-foundation/open-msupply/assets/5456533/0f4d7a96-3343-4859-a207-10e82d964611">

Reply:
actually.. was just following pattern but now you mentioned it it should be .1, .2, .3 not sure why the backend tests compare 0.1 > 0.01...